system/Session/Session.php

```diff
@@ -181,9 +190,15 @@ public function __construct(\SessionHandlerInterface $driver, $config)
 		$this->sessionTimeToUpdate      = $config->sessionTimeToUpdate;
 		$this->sessionRegenerateDestroy = $config->sessionRegenerateDestroy;
 
-		$this->cookieDomain = $config->cookieDomain;
-		$this->cookiePath   = $config->cookiePath;
-		$this->cookieSecure = $config->cookieSecure;
+		$this->cookieDomain   = $config->cookieDomain;
+		$this->cookiePath     = $config->cookiePath;
+		$this->cookieSecure   = $config->cookieSecure;
+		$this->cookieSameSite = $config->cookieSameSite;
```

Comment:
need default fallback value:

```suggestion
              $this->cookieSameSite = $config->cookieSameSite ?? $this->cookieSameSite;
```